docs/course_materials/exercises/05_Control_structures/homework_if.md

```diff
@@ -4,62 +4,33 @@
 ## if & else
 
 ### even_or_odd.py
+Create a file *even_or_odd.py*
```

Comment:
Replace the order of the lines:
new line
Create a file...

Reply:
done.